app/helpers/articles_helper.rb

```diff
@@ -1,4 +1,6 @@
 module ArticlesHelper
```

Comment:
**suggestion:** Instead of putting this into the helper, maybe use `ArticleDecorator`. I much prefer to see `@article.long_markdown?` instead of `long_markdown?(@article)` in views.